### ISSUE TITLE
B005 false positive

### ISSUE THREAD

**Issue by henryiii** [charliermarsh/ruff#2958]
The following code works in flake8-bugbear but triggers a false positive in Ruff:

```python
line = ""
line.lstrip("\ufeff")
```

```
B005 Using `.strip()` with multi-character strings is misleading the reader
```

It's incorrectly thinking that "\ufeff" is more than one character.

The B005 flag is active, and tested on 0.0.240 & 0.0.246.

**Comment by ortem** [comment]
This looks like a duplicate of already fixed #2851

**Comment by charliermarsh** [comment]
Yeah I just tested and it _seems_ like this is fixed on main.
